src/Services/Masa.Dcc.Service/Domain/App/Services/ConfigObjectDomainService.cs

```diff
@@ -225,6 +227,63 @@ private async Task CloneConfigObjectsAsync(List<AddConfigObjectDto> configObject
             await _configObjectRepository.AddRangeAsync(cloneConfigObjects);
         }
 
+        /// <summary>
+        /// cloning, check config object whether already exists.
+        /// </summary>
+        /// <param name="configObjects"></param>
+        /// <param name="appId">the 'appId' argument may be is 'application id' or 'public config Id' or 'biz config id'</param>
```

Comment:
Why appId can be or many times.

Reply:
@doddgu  the clone method is general. it's may be clone app's configuration or public's configuration or biz's configuration. so the appId is many ppssibilities in Check method